Formula/python@3.9.rb

```diff
@@ -155,7 +155,8 @@ def install
     # even if homebrew is not a /usr/local/lib. Try this with:
     # `brew install enchant && pip install pyenchant`
     inreplace "./Lib/ctypes/macholib/dyld.py" do |f|
-      f.gsub! "DEFAULT_LIBRARY_FALLBACK = [", "DEFAULT_LIBRARY_FALLBACK = [ '#{HOMEBREW_PREFIX}/lib',"
```

Comment:
The variable `DEFAULT_LIBRARY_FALLBACK` in dyld.py already contains `/usr/local/lib` [1] and I have checked all commit changed related to the file `Lib/ctypes/macholib/dyld.py` in cpython repo. Path `/usr/local/lib` is introduced since Python 3. So I think there is no need to add duplicate path at first. After I see the comment above inreplace method call which said

```
    # Allow python modules to use ctypes.find_library to find homebrew's stuff
    # even if homebrew is not a /usr/local/lib. Try this with:
    # `brew install enchant && pip install pyenchant`
```

 I would recommend that we should add `/usr/local/lib` to the variable `DEFAULT_LIBRARY_FALLBACK` only if `HOMEBREW_PREFIX` is not `/usr/local`. So performance of looking library should be faster when the library is missing in `/usr/local/lib`.

Second, I think user's library in home directory (e.g. `/home/user/lib`) should take first place than `/usr/local/lib`. 

```
DEFAULT_LIBRARY_FALLBACK = [
    os.path.expanduser("~/lib"),
    "/usr/local/lib",
    "/lib",
    "/usr/lib",
]
```

Any thought? if all goods, I'll revised my changes.

[1]: https://github.com/python/cpython/blob/bb70b2afe39ad4334a9f3449cddd28149bd628b6/Lib/ctypes/macholib/dyld.py#L29-L34

Reply:
What we have is robust, working, and simple (maintainable). I'd prefer keeping it that way, because if we change this we cannot rebuild all 454 python-based formulas to check. So let's go for "minimally-invasive" change.

Reply:
Okay, fair enough.